devops/continuous_integration/project_setup.rst

```diff
@@ -0,0 +1,91 @@
+Project setup
+=============
+
+The code necessary for this tutorial is found in the ``examples2`` repo, clone it and 
+move to the folder:
+
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/devops/ci/game
+
+
+Server repositories setup
+-------------------------
+
+We need 3 different repositories in the same server. Make sure to have an Artifactory running and available. You can download the free  `JFrog Artifactory Community Edition (CE) <https://conan.io/downloads.html>`_ and run it in your own computer. If you have another available Artifactory, it can be used too if you can create new repositories there. Log into the web UI and create 3 different local repositories called ``develop``, ``packages`` and ``products``.
+
```

Comment:
I think it would be nice to add here the lines to launch Artifactory CPP from Docker (in a version that works), so that users that are a bit experienced with using it do not have to move to other place to get the line.